styles.css

```diff
@@ -64,4 +64,13 @@
 
 .bbb_index_form {
     display: inline-block;
-}
\ No newline at end of file
+}
+
+.bigbluebuttonbn-guestaccess-view .warning{
```

Comment:
See the other comment, these css should not be needed, warnings should use the standard renderer:

$OUTPUT->notification()